tests/Console/ConfigurationResolverTest.php

```diff
@@ -961,6 +961,26 @@ public function provideResolveRulesCases()
                     '@A' => true,
                 ),
             ),
+            '@A - @C' => array(
+                array(
+                    'AB' => true,
+                    'AC' => 'a',
+                ),
+                array(
+                    '@A' => true,
+                    '@C' => false,
+                ),
+            ),
+            '@A - @D' => array(
```

Comment:
@pilgerone , go for main discussion:
https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/2545#discussion_r107527563

it has more complicated cases.